test/suites/dev-tanssi/staking/test_staking_rewards_balanced.ts

```diff
@@ -134,8 +134,8 @@ describeSuite({
                 const events = await polkadotJs.query.system.events();
                 const reward = await fetchRewardAuthorOrchestrator(events);
 
-                // 20% plus 1 from rounding
-                const collatorPercentage = (20n * reward.balance.toBigInt()) / 100n + 1n;
```

Comment:
This +1 is not needed after initializing the staking account with the existential deposit, not sure why because it is only +1, not +1_000_000_000 which is the ED